src/git-unix/git_unix.ml

```diff
@@ -13,13 +13,20 @@
  * ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
  * OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  *)
-module Net = Net
-module SHA1 = Git.Hash.Make(Digestif.SHA1)
-module Sync = Git.Sync.Make(Net)
-module HTTP = Http.Make
 
-module FS = struct
-  include Git.Store.FS(SHA1)(Fs)(Git.Inflate)(Git.Deflate)
+module Fs    = Fs
+module Net   = Net
+module SHA1  = Git.Hash.Make(Digestif.SHA1)
+module Sync  = Git.Sync.Make(Net)
+module Http  = Http.Make
+module Index = Index
+
+module Store = struct
+  include Git.Store.Make(SHA1)(Fs)(Git.Inflate)(Git.Deflate)
+
+  let with_fs ?root ?dotgit ?compression ?buffer fs =
```

Comment:
Could you merge `with_fs` with `create` (and call the new function `v`) and have `?(fs = Fs.v ?temp_dir ())` ?